### ISSUE TITLE
dropbear warning on every connection

### ISSUE THREAD

**Issue by shenki** [openbmc/openbmc#1340]
We get this message when connecting:

```
Feb 22 06:21:40 witherspoon dropbear[1073]: Failed loading /etc/dropbear/dropbear_dss_host_key
Feb 22 06:21:40 witherspoon dropbear[1073]: Failed loading /etc/dropbear/dropbear_ecdsa_host_key
```
The messages fill up the journal over time, pushing out more interesting information. We should fix or suppress the warning.

**Comment by mdmillerii** [comment]
We seem to have a host rsa key but not the others.

**Comment by hramasub** [comment]
dropbear is started with the wrong (/var/lib/...) rsa key path even while the path in the service file points to /etc/dropbear/.

My guess is that this happens since /etc/ is not mounted during systemd start, thus causing systemd to fallback to the default path?

Should verify if this issue goes away with [Eddies' patch to mount /etc/ before systemd start](https://gerrit.openbmc-project.xyz/#/c/7822/2) 

[Service]
Environment="**DROPBEAR_RSAKEY_DIR**=**/etc/dropbear**"
ExecStart=-/usr/sbin/dropbear -i **-r ${DROPBEAR_RSAKEY_DIR}/dropbear_rsa_host_key** $DROPBEAR_EXTRA_ARGS

**Comment by mdmillerii** [comment]
There is another copy of this issue that has what is required to remove these warnings.

We need to create 3 keys and pass them all to dropbear.

**Comment by geissonator** [comment]
Same as #1998 - I'll buy someone a beer if they fix this... it clogs up our journal when running automated testing (which does lots and lots of ssh connections).

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/9005> dropbear: Create rsa host key in user specified path vs /etc/dropbear
Resolves  #1340 dropbear warning on every connection
